Marlin/src/pins/lpc1769/pins_BTT_SKR_E3_TURBO.h

```diff
@@ -0,0 +1,266 @@
+#endif
+
+//
+// Filament Runout Sensor
+//
+#define FIL_RUNOUT_PIN                     P1_26  // E0DET
+#define FIL_RUNOUT2_PIN                    P1_25  // E1DET
+
+//
+// Power Supply Control
+//
+#ifndef PS_ON_PIN
+  #define PS_ON_PIN                        P1_21
+#endif
+
+// LED driving pin
+#define NEOPIXEL_PIN                       P1_24
+
+//
+// Power Loss Detection
+//
+#ifndef POWER_LOSS_PIN
+  #define POWER_LOSS_PIN                   P1_20  // PWRDET
+#endif
+
+//
+// Steppers
+//
+#define X_STEP_PIN                         P1_04
+#define X_DIR_PIN                          P1_08
+#define X_ENABLE_PIN                       P1_00
+#ifndef X_CS_PIN
+  #define X_CS_PIN                         P1_01
+#endif
+
+#define Y_STEP_PIN                         P1_14
+#define Y_DIR_PIN                          P1_15
+#define Y_ENABLE_PIN                       P1_09
+#ifndef Y_CS_PIN
+  #define Y_CS_PIN                         P1_10
+#endif
+
+#define Z_STEP_PIN                         P4_29
+#define Z_DIR_PIN                          P4_28
+#define Z_ENABLE_PIN                       P1_16
+#ifndef Z_CS_PIN
+  #define Z_CS_PIN                         P1_17
+#endif
+
+#define E0_STEP_PIN                        P2_06
+#define E0_DIR_PIN                         P2_07
+#define E0_ENABLE_PIN                      P0_04
+#ifndef E0_CS_PIN
+  #define E0_CS_PIN                        P0_05
+#endif
+
+#define E1_STEP_PIN                        P2_11
+#define E1_DIR_PIN                         P2_12
+#define E1_ENABLE_PIN                      P0_21
+#ifndef E1_CS_PIN
+  #define E1_CS_PIN                        P0_22
+#endif
+
+#if HAS_TMC_UART
+  /**
+   * TMC2208/TMC2209 stepper drivers
+   *
+   * Hardware serial communication ports.
+   * If undefined software serial is used according to the pins below
+   */
+
+  //
+  // Software serial
+  //
+  #define X_SERIAL_TX_PIN                  P1_01
+  #define X_SERIAL_RX_PIN                  P1_01
+
+  #define Y_SERIAL_TX_PIN                  P1_10
+  #define Y_SERIAL_RX_PIN                  P1_10
+
+  #define Z_SERIAL_TX_PIN                  P1_17
+  #define Z_SERIAL_RX_PIN                  P1_17
+
+  #define E0_SERIAL_TX_PIN                 P0_05
+  #define E0_SERIAL_RX_PIN                 P0_05
+
+  #define E1_SERIAL_TX_PIN                 P0_22
+  #define E1_SERIAL_RX_PIN                 P0_22
+
+  // Reduce baud rate to improve software serial reliability
+  #define TMC_BAUD_RATE                    19200
+#endif
+
+//
+// TMC Low Power Standby pins
+//
+#define X_STDBY_PIN                        P3_26
+#define Y_STDBY_PIN                        P3_25
+#define Z_STDBY_PIN                        P1_18
+#define E0_STDBY_PIN                       P1_19
+#define E1_STDBY_PIN                       P2_13
+
+//
+// Temperature Sensors
+//
+#define TEMP_0_PIN                         P0_24
+#define TEMP_1_PIN                         P0_23
+//#define TEMP_2_PIN                       P1_30   // Onboard thermistor
```

Comment:
Onboard thermistor is disabled by default since Marlin doesn't support a case/motherboard/ambient temp sensor yet. It *could* be classified as a chamber, but I'll leave it up to users who want to go that route.